apps/src/gamelab/PropTypes.js

```diff
@@ -54,6 +54,7 @@ export const AnimationKey = React.PropTypes.string;
  * @property {?string} sourceUrl
  * @property {Vector2} frameSize
  * @property {number} frameCount
+ * @property {bool} looping
```

Comment:
Nit throughout, let me know what you think - I'd use `isLooping` instead of `looping` to imply the boolean.  In this case I'm not sure it matters much, I can't imagine what else `looping` would be - but we should settle on one before we introduce this to serialized data.